packages/block-editor/src/components/block-list/block.js

```diff
@@ -269,6 +273,13 @@ function BlockListBlock( {
 			toggleSelection={ toggleSelection }
 		/>
 	);
+
+	// For aligned blocks, provide a wrapper element so the block can be
+	// positioned relative to the block column.
```

Comment:
Can we clarify in the comment that this is done using the className to avoid removing it by accident.